server/src/app.ts

```diff
@@ -0,0 +1,44 @@
+import express, { Express, NextFunction, Request, Response } from 'express';
+import dotenv from 'dotenv';
+import morgan from 'morgan';
+import cors from 'cors';
+// import { router as demoRouter } from './api/routes/demo.route';
+import { router as projectRouter } from './api/routes/project.route';
+
+dotenv.config();
+
+const app: Express = express();
+
+app.use(morgan('dev'));
+app.use(express.urlencoded({
+  extended: false
+}));
+app.use(express.json());
+app.use(cors({ origin: [`http://localhost:${process.env.CLIENT_APP_PORT}`, `${process.env.CLIENT_APP_URL}`] }));
+
+//====== Use Routers =======
```

Comment:
I see that you removed the appRouter in the app.ts. You also removed the app route and controller files. Please add it back. That is like the "Home page" for our API. Instead of "App Works", you can use the message "Code Collabo Web app" or something @samuko-things 

Do this, and I will take over the clean up from there.